### ISSUE TITLE
Full multiplayer compatibility

### ISSUE THREAD

**Issue by phantomeye1931** [snowy-shack/PortalMod#25]
- [x] Cubes get stuck and remain in place on client side but cannot be moved (I don't recall this)

- [x] Chamber sign packets
- [x] Cube holding?
- [x] Faith plate target selecting
- [ ] Dedicated servers

**Comment by phantomeye1931** [comment]
@LarsMans64 do you think you can work on this?

**Comment by LarsMans64** [comment]
ok, do you have some list of everything we know doesnt work?

**Comment by phantomeye1931** [comment]
- things

**Comment by phantomeye1931** [comment]
I know there are some issues with chamber sign packets, but not much more.

This needs a lot more testing

**Comment by phantomeye1931** [comment]
Most imporantly, dedicated servers simply don't boot right now, seems to be related to PacketInit

**Comment by phantomeye1931** [comment]
Ya @LarsMans64 do you think you could do this

**Comment by SerialSniper** [comment]
Last point is another issue and a much broader one

**Comment by phantomeye1931** [comment]
@SerialSniper yes but it needs to be done, and making an issue out of that specifically seems pointless. This issue is just that + some smaller fixes

**Comment by SerialSniper** [comment]
I didn't imply wanting to separate it, just not trivialize it

**Comment by phantomeye1931** [comment]
Closing this as it's stable enough to be split into specific issues
